portfolio-edit/edit-project.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+include './pullPortfolio.php';
+if (empty($_GET['id'])) {
+    echo 'unexpected error no id set';
+}else {
+    $id = $_GET['id'];
+    $projects = pullPortfolio();
```

Comment:
this is not the best way of getting 1 project, it gets all of them and then filters. can we have a new function that just gets the one project you need